app/components/attachment_button.js

```diff
@@ -65,12 +65,64 @@ export default class AttachmentButton extends PureComponent {
         intl: intlShape.isRequired,
     };
 
+    getPermissionDeniedMessage = (source) => {
+        const {formatMessage} = this.context.intl;
+        switch (source) {
+        case 'camera':
+            return {
+                title: formatMessage({
+                    id: 'mobile.android.camera_permission_denied_title',
+                    defaultMessage: 'Camera access is required',
+                }),
+                text: formatMessage({
+                    id: 'mobile.android.camera_permission_denied_description',
+                    defaultMessage: 'To take photos and videos with your camera, please change your permission settings.',
+                }),
+            };
+        case 'storage':
+            return {
+                title: formatMessage({
+                    id: 'mobile.android.storage_permission_denied_title',
+                    defaultMessage: 'File Storage access is required',
+                }),
+                text: formatMessage({
+                    id: 'mobile.android.storage_permission_denied_description',
+                    defaultMessage: 'To upload images from your Android device, please change your permission settings.',
```

Comment:
we need to update this text as is explicitly mention `Android device`. also lets change the id and remove `android` from it.

once you do the changes to the translations run `make i18n-extract` to get them extracted for you and the previous ones will also be removed.

Reply:
Please update your permission settings to upload files from your device.